.travis.yml

```diff
@@ -3,7 +3,7 @@ language: python
 cache: pip
 
 python:
-  - 3.6
+  - 3.6.2
```

Comment:
`fastapi` did not really want to install with less than 3.6.2.. perhaps we can drop some of these older python versions soon?

Reply:
I think we can drop `3.6`. It has reached EOL:
https://endoflife.date/python